### PR TITLE
fix flaky unit test

### PR DESCRIPTION
**Pull request by 315157973**
### Motivation
Topic level policy cache will only trigger initialization when `OwnedNamespaceBundle`. The following unit tests do not create producers or consumers, and just wait through sleep.
The cache will not initialize no matter how long it wait. If the cache is called without initializing, an error of 500 will be reported
So I created a producer or consumer for these tests, and avoided sleep

### CODE REVIEW

**Review by eolivelli — APPROVED** [review]
LGTM
I was going down this way in this commit
https://github.com/apache/pulsar/pull/8590/commits/624441ffb3ee4420d376916a54309d3954738e5f

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiOffloadTest.java [hunk 1]
```diff
@@ -180,43 +183,32 @@ public void testOffloadPolicies() throws Exception {
     public void testOffloadPoliciesApi() throws Exception {
         final String topicName = testTopic + UUID.randomUUID().toString();
         admin.topics().createPartitionedTopic(topicName, 3);
-        //wait for server init
-        Thread.sleep(1000);
+        pulsarClient.newProducer().topic(topicName).create().close();
+        Awaitility.await().atMost(10, TimeUnit.SECONDS)
```
Comment (eolivelli) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiOffloadTest.java:187]: please consider leaving the default wait time (here and below), that is 10 seconds

as @lhotari suggested in #9258

Comment (315157973) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiOffloadTest.java:187]: Okay, let me extend the timeout period, but based on experience, the initialization will be successful within 2 seconds.

**Review by sijie — APPROVED** [review]

**Review by jiazhai — APPROVED** [review]

**Comment by eolivelli** [comment]
@315157973 your error is this one
```
~/work/pulsar/pulsar/pulsar-client-all
     [java] The following classes needed for analysis were missing:
     [java]   jline.console.completer.Completer
Error:  Failed to execute goal on project pulsar-io-hbase: Could not resolve dependencies for project org.apache.pulsar:pulsar-io-hbase:jar:2.8.0-SNAPSHOT: Failed to collect dependencies at org.apache.hbase:hbase-server:jar:2.3.0 -> org.glassfish.web:javax.servlet.jsp:jar:2.3.2 -> org.glassfish:javax.el:jar:3.0.1-b06-SNAPSHOT: Failed to read artifact descriptor for org.glassfish:javax.el:jar:3.0.1-b06-SNAPSHOT: Could not transfer artifact org.glassfish:javax.el:pom:3.0.1-b06-SNAPSHOT from/to spring-plugins-release (https://repo.spring.io/plugins-release/): Authentication failed for https://repo.spring.io/plugins-release/org/glassfish/javax.el/3.0.1-b06-SNAPSHOT/javax.el-3.0.1-b06-SNAPSHOT.pom 401 Unauthorized -> [Help 1]
```

I guess we should commit #9278 and your error will go away
cc @codelipenghui @zymap

**Comment by eolivelli** [comment]
I have cherry picked this patch to branch-2.7

**Pull request merged**